src/me/StevenLawson/TotalFreedomMod/Listener/TFM_PlayerListener.java

```diff
@@ -923,4 +925,20 @@ public void onPlayerLogin(PlayerLoginEvent event)
     {
         TFM_ServerInterface.handlePlayerLogin(event);
     }
+    
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onBreakBlock(BlockBreakEvent event)
+    {
+        final Player player = event.getPlayer();
+        
+        if (player.getGameMode().equals(SURVIVAL))
+        {
+            for (ItemStack stack : event.getBlock().getDrops())
+            {
+                event.getBlock().setType(Material.AIR);
```

Comment:
No need to set the material type to air here. (It's a block break event, this is the default behavior.)